lib/api/controllers/securityController.js

```diff
@@ -0,0 +1,131 @@
+var
+  BadRequestError = require('../core/errors/badRequestError'),
+  ResponseObject = require('../core/models/responseObject');
+
+module.exports = function SecurityController (kuzzle) {
+
+  /**
+   * Get a specific role according to the given id
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.getRole = function (requestObject) {
+    kuzzle.pluginsManager.trigger('data:getRole', requestObject);
+
+    return kuzzle.repositories.role.loadRole(
+      kuzzle.repositories.role.getRoleFromRequestObject(requestObject)
+      )
+      .then(role => {
+        return Promise.resolve(new ResponseObject(requestObject, role || {}));
```

Comment:
Ok so, what do we stick to? I've seen some comments on #123 telling that promised returns should be harmonized on `q(response)`.